Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/DiscoverBounds.dfy

```diff
@@ -80,13 +80,13 @@ predicate LessThanFour(x: int) {
 
 method EnumerateOverInfiniteCollections() {
   // ===== iset
-  
   var s := {3, 3, 3, 5};
 
   // Once, the following RHS caused "u" to be auto-ghost. (Oddly enough, when using the same RHS as a
   // separate assignment, the RHS was not considered to be ghost. So, we test both here.)
   var u := iset x | x in s;
   u := iset x | x in s;
+  assert 3 in u;
```

Comment:
Do we have a sense of how many examples like this one will be affected, and do we want to explain the fix in the release notes?

Also, is this a case where we could rewrite the formula?  `x :| x in s` really just requires `s != {}`.

Reply:
I haven't done any measurements, but I don't think there will be many.

Note that the `assert` in the test file is needed for the `var y :| y in u && LessThanFour(y);` that follows. It would also have been fine to use `assert LessThanFour(3);`. Before this PR, I suspect the assertion was not needed because the `y :| ...` had triggered on every integer.

Regarding the possible rewrite, it is not needed here, because both `s != {}` and `|s| == 0` are enough to automatically prove the existence of an `x` in `x :| x in s`.